Server/Pages/Shipment/RegisterShipment.razor

```diff
@@ -0,0 +1,31 @@
+@using Repositories.Entities
+@using Repositories.Interfaces
+
+@code {
+
+    [Inject]
+    public ISenderRepository SenderRepository { get; set; }
+    [Inject]
+    public IPackageRepository PackageRepository { get; set; }
```

Comment:
Why are those public? why do they need GetSetters?

Reply:
Probably because of [Inject] access, but maybe it could be private. set is for inject I think, get is so you could access the injected object.